.tekton/discovery-server-push.yaml

```diff
@@ -0,0 +1,619 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/quipucords/quipucords?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch
+      == "main"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: discovery
+    appstudio.openshift.io/component: discovery-server
+    pipelines.appstudio.openshift.io/type: build
+  name: discovery-server-on-push
+  namespace: discovery-tenant
+spec:
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/redhat-user-workloads/discovery-tenant/discovery/discovery-server:{{revision}}
+  - name: dockerfile
+    value: Containerfile
+  - name: path-context
+    value: .
+  - name: hermetic
+    value: "true"
+  - name: build-source-image
+    value: "true"
+  # TODO: once proper cachi2-rust support is implemented, remove "allow_binary"
+  - name: prefetch-input
+    value: '[{"type": "pip", "allow_binary": "true"}, {"type": "rpm", "path": "dependencies"}]'
```

Comment:
this config allows using pip wheels instead of sdists, mitigating our issue with rust